pkg/machines/scripts/create_machine.sh

```diff
@@ -13,12 +13,25 @@ vmExists(){
 }
 
 endIfFailed(){
-	if [ $1 -ne 0 ]; then
-	    rm -f "$XMLS_FILE"
+    if [ $1 -ne 0 ]; then
```

Comment:
this if is not necessary here anymore. Also the name of the function could be something like `finish()` now

Reply:
Good point. Fixed.